.github/workflows/build.yml

```diff
@@ -145,17 +145,15 @@ jobs:
       - name: Setup dotnet tools
         run: dotnet tool restore
 
-      # - name: Check F# formatting (fantomas)
-      #   run: dotnet fantomas src/Fable.Transforms/Python --check
-
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Install poetry
         run: |
-          pip install poetry
+          pipx install poetry
```

Comment:
Is `pipx` available out of the box or is it something we need to install?

If this is the latest, we need to update [Dockerfile](https://github.com/fable-compiler/Fable/blob/5a59e3a06e26902dbb6b6f14ebca83ed3fe54ca8/.devcontainer/Dockerfile)